deploy/olm-catalog/gitops-operator/manifests/gitops-operator.clusterserviceversion.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    containerImage: quay.io/redhat-developer/gitops-backend-operator:v0.0.1
+    containerImage: quay.io/cbanavik/gitops-backend-operator:dep
```

Comment:
I don't think this should be in this PR?

Reply:
This is an outdated commit. It was changed earlier